src/components/feedback-form.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+// import Form from "./form";
+import styles from "./call-to-action.module.css";
+import PropTypes from "prop-types";
+import Form from "./form";
+
+
+function FeedbackForm() {
+
+  const additionalFormAttr = {
+    "data-netlify": "true",
+    method: "post"
+  }
```

Comment:
Suggestion: Remove this variable declaration outside of the component. It doesn't change so we don't need to recreate it on rerenders.

Reply:
Do you think that this would be better to have at props?